config/locales/en.yml

```diff
@@ -526,6 +526,9 @@ en:
     staircase_bought_seems_high: "You said %{percentage}% was bought in this staircasing transaction, which seems high. Are you sure?"
     monthly_charges_over_soft_max:
       title_text: "The amount of monthly charges is high for this type of property and sale type"
+    discounted_sale_value:
+      title_text: "Mortgage, deposit, and grant total must equal £%{value_with_discount}"
+      informative_text: "Your given mortgage, deposit and grand total is £%{mortgage_deposit_and_grand_total}"
```

Comment:
I'm not sure the grammar of this sentence is that clear - given as in it's been input by the user? More of a copy question but there feels like a better way of saying this

Reply:
You're right, I'll get it PO reviewed and will see if any comments come out of it!